assets/src/design-system/components/input/index.js

```diff
@@ -65,7 +65,7 @@ const InputContainer = styled.div(
     justify-content: space-between;
     height: 36px;
     padding: 4px 12px;
-    background-color: ${theme.colors.bg.primary};
+    background-color: transparent;
```

Comment:
I think @miina did this elsewhere but needed it here too for things to look right.